### DISCUSSION TITLE
Separate exclusion areas in the same page

### DISCUSSION THREAD

**Discussion by SierraJn** [red6/pdfcompare#113] [Q&A] [answered]
I am using the library and the exclusions json file generated for each page is the maximum rectangle containing all the differences.

Does it exist a way to generate different rectangle (one per each difference) in the json file? In this way we could check more information.

Current:
![image](https://user-images.githubusercontent.com/11092863/138438418-97cf21d2-c746-4f10-895c-66d6cceddf7e.png)

Is that possible?:
![image](https://user-images.githubusercontent.com/11092863/138438833-b558ed00-0150-49b3-a385-ff7010c2a87f.png)

**Comment by finsterwalder** [comment, marked as answer]
It's not possible to generate those multiple smaller exclusions automatically.
You can define as many exclusions as you want. But you would have to do that "manually".
There exists a UI, however, that allows to define exclusions by drag and drop.
See the README how to start the UI.

**Reply by SierraJn** [reply]
Thank you for your reply, but I cannot execute it: I am getting the message: no main manifest attribute.

**Reply by finsterwalder** [reply]
What exactly did you execute?
java -jar pdfcompare-1.1.60-full.jar
Works for me.

**Reply by SierraJn** [reply]
Anything went wrong. After compile again it works!.

Thank you for your help.
